src/olympia/devhub/templates/devhub/suppressed_email.html

```diff
@@ -0,0 +1,8 @@
+{% if request.suppressed_email and waffle.switch("suppressed-email") %}
+<div class="notification-box warning" id="suppressed-email">
+    <p>
+        We have discovered that your email "{{request.user.email}}" is unable to recieve emails from us. Please update your email address to one that can recieve emails from us.
```

Comment:
I have no idea how to localize this string.

Reply:
```suggestion
        {% trans email=request.user.email %}
            We have discovered that your email "{{ email }}" is unable to receive emails from us. Please update your email address to one that can receive emails from us.
        {% endtrans %}
```

Reply:
Thanks